server/routerlicious/packages/protocol-base/src/protocol.ts

```diff
@@ -159,14 +159,17 @@ export class ProtocolOpHandler implements IProtocolHandler {
 
 		if (scrubUserData) {
 			// In place, remove any identifying client information
-			snapshot.members.forEach((member) => {
-				member[1] = {
-					...member[1],
-					client: {
-						...member[1].client,
-						user: { id: "" },
+			snapshot.members = snapshot.members.map(([id, sequencedClient]) => {
+				return [
```

Comment:
Just nit, maybe I am wrong(following should work for object but I assume the same for array) I think you can do 
```
([id,
					{
						...sequencedClient,
						client: {
							...sequencedClient.client,
							user: { id: "" },
						},
					},])
```

Reply:
True, I'll clean this up.